stacks/cli/src/commands/function/index.ts

```diff
@@ -0,0 +1,15 @@
+import {Namespace, NamespaceMetadata, NamespaceMap} from "@ionic/cli-framework";
+import {DependencyNamespace} from "./dependency";
+
+export class FunctionNamespace extends Namespace {
```

Comment:
You should extend this namespace from  SpicaNamespace which contains all the things needed for a namespace.
```suggestion
import {SpicaNamespace} from "../../interface";

export class FunctionNamespace extends SpicaNamespace {
```